email_finder.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+text_file = input("File name: ")
+with open(text_file, 'r') as file:
+    text = file.read()
+
+    print(re.findall(r'[a-zA-Z0-9_.]+@(?:[a-zA-Z0-9-]+\.)+[a-zA-z]{2,}', text)) # any other symbols relevant for email address can be added in []
```

Comment:
The task was to replace emails. You shouldn't print them all

Reply:
Just forgot to remove print lines from the code before submitting (used it to test the code)